staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -126,12 +126,22 @@ type JobSpec struct {
 	// TTLAfterFinished feature.
 	// +optional
 	TTLSecondsAfterFinished *int32 `json:"ttlSecondsAfterFinished,omitempty" protobuf:"varint,8,opt,name=ttlSecondsAfterFinished"`
+
+	// Suspend specifies whether the Job controller should create Pods or not. If
+	// a Job is created with suspend set to true, no Pods are created by the Job
+	// controller. If a Job is suspended after creation (i.e. the flag goes from
+	// false to true), the Job controller will delete all active Pods associated
+	// with this Job. Users must design their workload to gracefully handle this.
+	// This is an alpha field and requires enabling the SuspendJob feature gate.
+	// Defaults to false.
+	// +optional
+	Suspend bool `json:"suspend,omitempty" protobuf:"varint,10,opt,name=suspend"`
 }
 
 // JobStatus represents the current state of a Job.
 type JobStatus struct {
-	// The latest available observations of an object's current state.
-	// When a job fails, one of the conditions will have type == "Failed".
+	// The latest available observations of an object's current state. Can be
+	// one of "Suspended", "Failed", or "Complete".
```

Comment:
Don't use wording that implies other conditions are not possible.

Reply:
I assume you meant "Don't use wording that implies other conditions are ~~not~~ possible" because these are the only three possible values. Updated to use "will be" instead of "can be" :+1:

Reply:
I don't think that's what he meant. The wording shouldn't imply that there won't be other conditions in the future.

Reply:
Right, what Aldo said. :)